ArduCopter/mode_rtl.cpp

```diff
@@ -335,6 +335,18 @@ void Copter::ModeRTL::land_start()
     auto_yaw.set_mode(AUTO_YAW_HOLD);
 }
 
+bool Copter::ModeRTL::is_landing() const
+{
+    switch(_state) {
+    case RTL_Land:
+    case RTL_FinalDescent:
```

Comment:
This is a good question, is final descent a landing?

Reply:
@peterbarker could you put RTL_FinalDescent in no landing ? after testing this in my company, we found this is better to not beeing flagged as landing

Reply:
Done.